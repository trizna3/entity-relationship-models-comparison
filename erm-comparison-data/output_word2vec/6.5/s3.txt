questions;options-questions;answers
users-users
tests-tests
CONTRACT_11_ASSOCIATION:E-questions-options
CONTRACT_11_ASSOCIATION:S-questions-answers
