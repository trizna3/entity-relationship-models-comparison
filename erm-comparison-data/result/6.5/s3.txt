Mapping 0/4 pairs, 0 additional pairs.
Missing pairs: 
 tests-tests
 questions-questions
 options-answers
 users-users
Additional pairs: -

Mapping  0/0 transformations, 0 additional transformations.
Missing transformations: -
Additional transformations: -
