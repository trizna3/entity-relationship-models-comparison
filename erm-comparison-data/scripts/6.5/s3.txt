entity set users {
	"name"
	"surname"
};

entity set tests {
	"is available"
};

entity set questions {
	"text"
};

entity set "test registration" {
	"date"
};

entity set answers {
	"text"
};

association between 
{
	tests MANY
	questions MANY
}{};

association between 
{
	tests ONE
	"test registration" MANY
}{};

association between 
{
	"test registration" MANY
	users ONE
}{};

association "has options" between 
{
	questions ONE
	answers ONE
}{};

association "has the right answer" between 
{
	questions ONE
	answers MANY
}{};

association "user answers" between 
{
	"test registration" MANY
	questions MANY
	answers MANY
}{};
