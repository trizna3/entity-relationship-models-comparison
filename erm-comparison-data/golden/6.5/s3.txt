users-users
tests-tests
questions-questions
options-answers